easybuild/easyconfigs/h/Horovod/Horovod-0.23.0-foss-2021a-PyTorch-1.10.0.eb

```diff
@@ -0,0 +1,47 @@
+easyblock = 'PythonBundle'
+
+name = 'Horovod'
+version = '0.23.0'
+local_pt_version = '1.10.0'
+versionsuffix = '-PyTorch-%s' % local_pt_version
```

Comment:
`versionsuffix` should include `-CUDA-%(cudaver)` and filename changed to reflect that addition. I'm not sure whether we want the CUDA before or after the PyTorch.

Reply:
Looking at `Horovod-0.22.1-foss-2021a-CUDA-11.3.1-TensorFlow-2.6.0.eb` I'll put `CUDA` first.